src/openrct2/actions/StaffFireAction.cpp

```diff
@@ -39,14 +39,14 @@ GameActions::Result StaffFireAction::Query() const
     if (_spriteId.ToUnderlying() >= MAX_ENTITIES || _spriteId.IsNull())
     {
         LOG_ERROR("Invalid spriteId. spriteId = %u", _spriteId);
-        return GameActions::Result(GameActions::Status::InvalidParameters, STR_NONE, STR_NONE);
+        return GameActions::Result(GameActions::Status::InvalidParameters, STR_ERR_INVALID_PARAMETER, STR_ERR_STAFF_NOT_FOUND);
```

Comment:
Isn't this one out of range?
```suggestion
        return GameActions::Result(GameActions::Status::InvalidParameters, STR_ERR_INVALID_PARAMETER, STR_ERR_VALUE_OUT_OF_RANGE);
```

Reply:
You are right, that is out of range. The "staff not found" message should only be used after we try to get an entity from the sprite. I pushed a fix